packages/js/api-client/tsconfig.prod.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["./tests/**", "./vitest.config.mts"]
+}
```

Comment:
This tsconfig is used when building types for the package. The excludes ensure that only the types appropriate for distribution are included in the package's type output.